packages/forms/src/UIForm/fieldsets/Array/Array.component.js

```diff
@@ -6,12 +6,16 @@ import defaultTemplates from '../../utils/templates';
 import defaultWidgets from '../../utils/widgets';
 
 function adaptKeyWithIndex(keys, index) {
-	let indexedKeys = keys;
-	const firstIndexPlaceholder = indexedKeys.indexOf('');
-	if (firstIndexPlaceholder >= 0) {
-		indexedKeys = [...keys];
-		indexedKeys[firstIndexPlaceholder] = index;
+	if (!keys) {
```

Comment:
This check could be made by callers before calling adaptKeyWithIndex to have a consistent return type from this method

Reply:
Thanks, fixed